samples/drivers/led_ws2812/boards/bbc_microbit.overlay

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2019, Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <dt-bindings/gpio/gpio.h>
+
+/ {
+	led_strip: ws2812 {
+		compatible = "worldsemi,ws2812-gpio";
+		label = "WS2812";
+
+		chain-length = <16>; /* arbitrary; change at will */
```

Comment:
Could we make this 24 since that's how many LEDs the Zip Halo has: https://www.kitronik.co.uk/5625-zip-halo-for-the-bbc-microbit.html

That would also match what `samples/drivers/led_ws2812/src/main.c` used to have with the _sw driver (now called gpio).

Reply:
Done, thanks. I like your new avatar btw.